sorc/ncep_post.fd/MISCLN.f

```diff
@@ -146,7 +146,7 @@ SUBROUTINE MISCLN
                                             UBND,   VBND,   RHBND,     &
                                             WBND,   T7D,    Q7D,       &
                                             U7D,    V6D,    P7D,       &
-                                            ICINGFD,GTGFD,CATFD,MWTFD
+                                            ICINGFD,GTGFD,CATFD,MWTFD,MIDCAL
```

Comment:
@EdwardColon-NOAA Please add a change log at the beginning section of this subroutine.